lib/dataset-tools/flags.js

```diff
@@ -19,6 +19,8 @@
 // Author: Giovanni Campagna <gcampagn@cs.stanford.edu>
 "use strict";
 
+const flagsMap = {'R':'replaced', 'P':'augmented', 'C':'contextual', 'S':'synthetic', 'E':'eval'};
```

Comment:
Add a comment that the order of entries is significant, because the order of flags is significant when parsing them.

(The order matters because there are multiwoz dialogues with ID that starts with "SNG" - but they should not get the "S" flag so we prefix the ID with "E", eval)